control_panel_api/models.py

```diff
@@ -45,6 +45,8 @@ def _slugify(name):
     slug = AutoSlugField(populate_from='name', slugify_function=_slugify)
     repo_url = models.URLField(max_length=512, blank=True, default='')
     created_by = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
+    users = models.ManyToManyField(
```

Comment:
On the subject of relationships (see other comment), do we need this association now? An app would have many `userapps`, objects containing app they're in and if they're an admin for the app.

In the serialiser will expose `userapps` which means the client will get the apps there (as it gets the S3 buckets via `users3buckets`) and this will be consistent with the other associations.